lib/openstax/rescue_from/wrapped_exception.rb

```diff
@@ -0,0 +1,76 @@
+module OpenStax
+  module RescueFrom
+    class WrappedException
+      attr_reader :exception
+
+      def initialize(exception)
+        @exception = exception
+      end
+
+      def header
+        @header ||= cause.blank? ? 'An exception occurred' : 'Exception cause'
```

Comment:
`cause.blank?` means that this WrappedException *HAS* a cause inside it (doesn't mean that it *IS* the cause) -- `cause` is the nested exception inside.  The tutor line you referenced prints "Exception cause" when the exception being logged *IS* the cause.  That make sense?